tests/pytest/core/test_models.py

```diff
@@ -232,8 +177,8 @@ def test_EligibilityType_enrollment_index_template():
 
 
 @pytest.mark.django_db
-def test_EligibilityType_enrollment_success_template():
-    new_eligibility_type = EligibilityType.objects.create()
+def test_EnrollmentFlow_enrollment_success_template():
+    new_eligibility_type = EnrollmentFlow.objects.create()
```

Comment:
`new_flow`

Reply:
renamed it [here](https://github.com/cal-itp/benefits/blob/903f76f7bf932cf401dba9e78887ee6b2da5c1b6/tests/pytest/core/test_models.py#L181)